src/jarabe/controlpanel/cmd.py

```diff
@@ -24,8 +24,8 @@
 
 _RESTART = 1
 
-_same_option_warning = _('sugar-control-panel: WARNING, found more than one'
-                         ' option with the same name: %s module: %r')
+_same_option_warning = _({'warning:sugar-control-panel: WARNING, reason:found more than one' 
```

Comment:
@nswarup14 Looks like there's whitespace issues in line 27 and 28. 

Bad review from my end. I'll fix it in a patch. @quozl should I squash it and force push with e04e441